exawind/OC6_platform_wave_tank/fixed_platform_irregular_waves/README.md

```diff
@@ -0,0 +1,11 @@
+This benchmark problem uses the two-phase ExaWind solver, where water and air are simulated using the volume-of-fluid approach in both Nalu-Wind and AMR-Wind. The setup conforms to load case 3.3 from Phase Ia of the OC6 project (the definition document can be accessed here: https://www.nrel.gov/docs/fy22osti/80803.pdf). This load case involves a fixed platform within an irregular wave field, where the simulation outputs are forces and moments on the platform. The reference data for this case are from experiments performed in the Concept Basin of the Maritime Research Institute Netherlands (MARIN) using a 1:50 scale platform model. The numerical simulations are also performed in model scale, whereas the results have been scaled up to full scale using Froude scaling, as is typical for the OC6 project.
+
+The AMR-Wind computational domain extends from -10m to 10m in the x direction, which is only a portion of the length of the original wave tank, while it extends from -2m to 2m in y, the full width of the tank, and from -3.6m to 2.4m in z, reaching the full depth of the tank and providing enough room around the vertical extent of the platform. Two levels of refinement are added, and these are targeted at the position of the air-water interface and the platform geometry. The refinements reach a resolution of 31.25mm in the x and y directions and 15.625mm in the z direction. The inlet in x uses a wave generation boundary condition, and the waves are forced into the domain with a relaxation zone. The outlet in x has a pressure outflow condition and the waves are absorbed using a numerical beach. The other boundaries are represented as slip walls.
+
+The Nalu-Wind computational mesh is tetrahedral and adheres to the mesh resolution recommendations in section 3.2.3 of this paper: https://www.sciencedirect.com/science/article/pii/S0029801822013580. (Need way to access mesh)
```

Comment:
The mesh files are now available at https://github.com/Exawind/exawind-benchmarks/releases/download/mesh_assets/oc5-resmatch.e.tar.gz and https://github.com/Exawind/exawind-benchmarks/releases/download/mesh_assets/modes_HOS_SWENSE.dat.bz2.  I might have to edit the `downloadmeshes.py` to account for them, but we can worry about that later.